### ISSUE TITLE
How to match host with wildcard name

### ISSUE THREAD

**Issue by aravindakumarbootlabs** [api7/lua-resty-radixtree#123]
I am trying to match only the host with wildcard name, no paths, methods required, how to do this with package, is there any alternative method that can be used.

**Comment by tzssangglass** [comment]
No. This lib is mostly a deep prefix match, and the path is required.
